### ISSUE TITLE
Error loading GBA ROM (MPK2)

### ISSUE THREAD

**Issue by mariofan12ify** [Kermalis/VGMusicStudio#88]
I keep getting this error whenever I open up certain GBA roms:

'Error parsing "MP2K.yaml"
Game code "blahblah" is missing.'

the "blahblah" part is different for each game.

Does this mean that the game isn't supported?

![image](https://user-images.githubusercontent.com/35483921/116452956-761ee480-a89d-11eb-81d8-daa862377ea7.png)

**Comment by Tulu473** [comment]
Correct, for now if you dont know how to manually add the games, post the name of them and I'm sure Kermalis will try to add them when they get a chance, as long as their sound engine is supported.
